### PR TITLE
Adding Velobike Moscow

### PR DESCRIPTION
**Pull request by R0nd**

### CODE REVIEW

**Comment by eskerda** [comment]
```
(v)pybikes git:R0nd-patch-1 ❯ sh runtests.sh TestDataFiles                                                                                                              ....
======================================================================
FAIL: test_instances (__main__.TestDataFiles)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/unittest_pybikes.py", line 231, in test_instances
    self._test_instance_unique(instance, schema)
  File "tests/unittest_pybikes.py", line 248, in _test_instance_unique
    self.fail(msg)
AssertionError: ("Tag velobike in {u'tag': u'velobike', u'meta': {u'latitude': 37.616667, u'city': u'Moscow', u'name': u'Velobike', u'longitude': 55.75, u'country': u'RU'}, u'feed_url': u'http://apivelobike.velobike.ru/ride/parkings'} from velobikeMSK.json is not unique. ", '\n Already being used by ', "{u'tag': u'velobike', u'meta': {u'latitude': 51.166667, u'city': u'Astana', u'name': u'Velobike', u'longitude': 71.416667, u'country': u'KZ'}, u'feed_url': u'http://velobike.kz/rest/app.php/stations?lang=en&_dc=1446191774843'} in velobike.json")

----------------------------------------------------------------------
Ran 1 test in 0.034s

FAILED (failures=1)
```

There's already a system called velobike, even though their implementation differ, so I guess they are different systems. 

Can you:
- rename `tag`  from `velobike` to `velobike-moscow`
- rename `VelobikeMSK.py` to `velobike_ru.py` (Since it's possible there might appear more systems in Russia using the same scheme)
- rename `velobikeMSK.json` to `velobike_ru.json`
- rename `system` to `Velobike RU`
- rename `company` to `ЗАО «СитиБайк»`

Thanks!

**Comment by R0nd** [comment]
Done! Thanks for the corrections.

File: pybikes/velobike_ru.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+import json
+
+from .base import BikeShareSystem, BikeShareStation
+from . import utils
+
+
+class velobike_ru(BikeShareSystem):
+
```
Comment (eskerda) [inline pybikes/velobike_ru.py]: `VelobikeRU`

File: pybikes/velobike_ru.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    sync = True
+
+    meta = {
+        'system': 'Velobike RU',
+        'company': 'ЗАО «СитиБайк»'
+    }
+
+    def __init__(self, tag, feed_url, meta):
+        super(velobike_ru, self).__init__(tag, meta)
```
Comment (eskerda) [inline pybikes/velobike_ru.py]: `VelobikeRU`

File: pybikes/data/velobike_ru.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+                "name": "Velobike",
+                "country": "RU",
+                "longitude": 55.75,
+                "latitude": 37.616667
+            },
+            "feed_url": "http://apivelobike.velobike.ru/ride/parkings"
+        }
+    ],
+    "system": "velobike",
+    "class": "velobike-ru"
```
Comment (eskerda) [inline pybikes/data/velobike_ru.json]: `"class": "VelobikeRU"`
`"system: "velobike_ru"`

File: pybikes/velobike_ru.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        #  "FreePlaces": 12,
+        #  "Id": "0415",
+        #  "IsLocked": true,
+        #  "Position": {
+        #    "Lat": 55.7914268,
+        #    "Lon": 37.5905396
+        # },
+        # "TotalPlaces": 12
+        #}
+        for item in data['data']:
```
Comment (eskerda) [inline pybikes/velobike_ru.py]: On my endpoint this call falls and items appear to be in `data['Items']`

**Comment by eskerda** [comment]
I've added some more comments. Sorry, naming conventions should be better and clearly explained on the documentation!

Also, there appears to be an issue whilst downloading feed. See comments

Thank you again!

**Comment by R0nd** [comment]
My bad, committed a new set of corrections.

**Comment by eskerda** [comment]
:+1: Looks good, thank you!

**Pull request merged**